### PR TITLE
Release v8.0.0

### PR DESCRIPTION
**Pull request by jeffbyrnes**
## Description

Release v8.0.0

## Issues Resolved

n/a

## Check List

- [x] All tests pass. See TESTING.md for details.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by Xorima** [comment]
Hey, can you fix up the mdl issues, we normally use `-` for lists

**Comment by jeffbyrnes** [comment]
No prob; my Markdown syntax thing in Sublime automatically alters nested lists to use `+` and `*`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,15 @@
...
-## Unreleased
-
+## 8.0.0 (2020-07-09)
+
+- Drop support for:
+  - Debian 8
+  - Ubuntu < 16.04
+  - Amazon Linux 1
+- Set default PHP version to 7.2.x
+- Update Ubuntu 18.04 support to source build PHP 7.2
```
Comment (ramereth) [inline CHANGELOG.md:12]: 7.2 is technically [EOL'd in November](https://www.php.net/supported-versions.php), we may want to default to 7.4 where possible.

Comment (jeffbyrnes) [reply CHANGELOG.md:12]: The reasoning behind this is that 18.04’s package installs 7.2, so all of the attributes are meant to align with that.

I hear you on the EOL coming up soon; any idea if 18.04 is gonna get a bump to its package versions?

**Comment by jeffbyrnes** [comment]
@Xorima list syntax has been updated!

File: CHANGELOG.md (hunk 1 above)
Comment (ramereth) [reply CHANGELOG.md:12]: Ah good point, then this should be OK for now. We can see what happens later I guess.

**Pull request merged**